### ISSUE TITLE
Aniyomi can't get any extension list 

### ISSUE THREAD

**Issue by sheikhwamik** [jmir1/aniyomi#875]
### Steps to reproduce

1. Open app
2. Go to browser option
3. Then go to Anime extensions
4. Try to reload the extensions

### Expected behavior

All anime extensions which are available should show up and can be downloaded or updated.

### Actual behavior

But whenever I open anime extensions, it shows "Failed to get extension list" even if i try to refresh the page.

### Crash logs

--------- beginning of system
--------- beginning of main
App version: 0.12.3.8 (standard, dc8d1af, 72, 2022-08-25T23:30Z)
Android version: 11 (SDK 30)
Android build ID: RKQ1.201004.002 test-keys
Device brand: xiaomi
Device manufacturer: Xiaomi
Device name: ginkgo
Device model: Redmi Note 8
Device product name: ginkgo

### Aniyomi version

aniyomi-v0.12.3.8.apk (114 MB)

### Android version

11 RKQ1.201004.002

### Device

Mi Note 8

### Other details

![Screenshot_2023-01-24-09-36-51-993_xyz jmir tachiyomi mi](https://user-images.githubusercontent.com/123435329/214210263-fc928d6c-d351-4fb6-aec0-10f85fc56569.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a short but informative title.
- [X] If this is an issue with an extension, I should be opening an issue in the [extensions repository](https://github.com/jmir1/aniyomi-extensions/issues/new/choose).
- [X] I have tried the [troubleshooting guide](https://aniyomi.org/help/guides/troubleshooting/).
- [X] I have updated the app to version **[0.12.3.8](https://github.com/jmir1/aniyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I will fill out all of the requested information in this form.

**Comment by Mitrajeet-Golsangi** [comment]
Yes the same issue is happening in the latest release as well. I've downloaded the stable v0.12.3.8

**Comment by Firefox2215e** [comment]
Yes I am also facing the problem in the stable v0.12.3.8

**Comment by Vishu-hiby69** [comment]
Same problem

**Comment by HelpMeGetANamePlease** [comment]
This is happening to me too.

**Comment by Drkwise** [comment]
> This is happening to me too. I tried installing the extension apk but it didn't show up in the app.

Same bro

**Comment by HelpMeGetANamePlease** [comment]
http://github.com/jmir1/aniyomi-mpv-beta/releases/latest
This beta version works.

**Comment by Drkwise** [comment]
How to fix the "failed to get manga extension list" in aniyomi?

**Comment by HelpMeGetANamePlease** [comment]
> 

https://github.com/jmir1/aniyomi-mpv-beta/releases/latest
This is the link to the beta version of Aniyomi that fixes the issue.

**Comment by JhDeLy-WB** [comment]
Como se arregla el error de "no se pudo descargar el listado de extensiones "

**Comment by JhDeLy-WB** [comment]
#900

**Comment by Quickdesh** [comment]
Duplicate of #844

**Comment by 2Hasnat** [comment]
Some of the extensions such as (aniyomi extension) can not get subtitles i don't know what should i do, if anyone knows how to solve this problem please reply it will be a great help thanks.

**Comment by Lokie-IND** [comment]
Guys TRY using PRIVATE DNS in your mobile settings. customize DNS and save it by (dns.adguard.com) and try browsing extentions.
